reporters/plain_reporter.py

```diff
@@ -11,9 +12,34 @@ def handle_message(self, msg):
         self._messages.append(msg)
 
     def print_message_ids(self):
+        # Sort the messages.
+        self.sort_messages()
+        for msg in self._messages:
+            code = msg.msg_id
+            print(code, '({}) {}\n    {}'.format(msg.symbol, msg.obj, msg.msg))
+
+    def sort_messages(self):
         # Sort the messages by their type.
         self._messages.sort(key=lambda s: s[0])
 
-        for msg in self._messages:
-            code = msg.msg_id
-            print(code, '({})\n    {}'.format(msg.symbol, msg.msg))
+        i = 0
+        while i < len(self._messages):
+            current_id = self._messages[i].msg_id
+            count = 1
```

Comment:
I use this variable to add the number of times this error appears in the message.